galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -366,3 +366,12 @@ messages:
         type: string
       - name: gateway
         type: string
+
+  - name: "ServiceEntryProtocolAndAddressesEmpty"
+    code: IST0133
+    level: Warning
+    description: "Addresses and Protocol is Empty in ServiceEntry."
+    template: "one or more Protocol empty in ServiceEntry %s with Addresses Empty."
```

Comment:
You shouldn't put the name of the ServiceEntry into the `template`.  The name should already be in the resource location.

Reply:
You are Report()ing one message per empty Protocol.  The message doesn't need to say "one or more Protocol empty...", it can just say "missing Protocol..."
I am not sure you need to say "with Addresses Empty".  There should be explanation text explaining the problem and how to fix it at https://istio.io/latest/docs/reference/config/analysis/ist0133/ , but the message should be concise and doesn't need to list exceptions.

Reply:
I think we need more info on *why* this is bad

Reply:
@esnible I can create PR for istio.io, we can put the information on the web.

Reply:
@howardjohn even this configuration is valid, since this can cause mesh problem, I believe we can put warning or info level probably?

Reply:
this is done
```
Warning [IST0133] (ServiceEntry service-entry-test-03.default galley/pkg/config/analysis/analyzers/testdata/serviceentry-empty-protocol-addresses.yaml:49) missing Protocol and Addresses.
Warning [IST0133] (ServiceEntry service-entry-test-04.default galley/pkg/config/analysis/analyzers/testdata/serviceentry-empty-protocol-addresses.yaml:65) missing Protocol and Addresses.
```